tests/integration_tests/performance/test_boottime.py

```diff
@@ -69,6 +69,31 @@ def find_events(log_data):
     return timestamps
 
 
+def get_systemd_analyze_times(microvm):
+    """
+    Parse systemd-analyze output
+    """
+    rc, stdout, stderr = microvm.ssh.run("systemd-analyze")
+    assert rc == 0, stderr
+    assert stderr == ""
+
+    boot_line = stdout.splitlines()[0]
+    pattern = r"Startup finished in (.*)\s+\(kernel\) \+ (.*)\s+\(userspace\) = (.*)\s*"
+    kernel, userspace, total = re.findall(pattern, boot_line)[0]
+
+    def to_ms(v):
+        if "ms" not in v:
+            return int(v[:-1]) * 1000
+        else:
+            return int(v[:-2])
+
+    kernel = to_ms(kernel)
+    userspace = to_ms(userspace)
+    total = to_ms(total)
+
+    return kernel, userspace, total
```

Comment:
Since these are the only numbers in the line, this works a is a bit more succinct:

```python
return [int(val) * (1000 if unit == 's' else 1) for val, unit in re.findall(r"(\d+)(s|ms)", boot_line)]
```